src/features/proxy/ws_history.md

```diff
@@ -1 +1,23 @@
-# WS History (WIP)
+# WS History
+
+## Streams
+
+The WS History feature allows you to view data exchange between the client and the server passing through the proxy.
+
+When an HTTP Connection is upgraded to Websocket, it will appear in the stream table.
+
+![WS History Streams](/_images/ws_streams.png)
+
+- Clicking on a stream will display the messages and their direction.
```

Comment:
Unsure about this dot for steps, maybe if the images were aligned with the text that would look ok?
You could try to set a width to the image (200 is an example):
```html
<img src="/_images/ws_streams.png" alt="Websocket History Streams" width="200"/>
```

Reply:
The image appears broken in the render, is it going to work anyway?